### DISCUSSION TITLE
What’s new in Budibase: January 2023 🎨🔗🖇️

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#9581] [Announcements]
Once again, we spent January improving the Budibase onboarding experience. We want users to experience the full power of Budibase as quickly as possible. We've also added a powerful new feature that allows you to reuse secrets and datasource credentials. Check out the notes below to learn more.

# New features ✨

### 🔀 Environment variables
**🔒 BUSINESS TIER**
With environment variables, you can reuse secrets and datasource credentials across different apps. They also allow you to develop on a staging database and only use a production database when the app is deployed.

For further information, please read the [Environment Variable docs](https://docs.budibase.com/docs/environment-variables).

![env-variables](https://user-images.githubusercontent.com/49767913/217480460-ee8bbe55-1c01-4ad3-b9ca-80c2f3fce357.png)

---

### 🛫 New onboarding experience
'You never get a second chance to make a first impression.' When users sign up for Budibase, they'll experience a new guided flow that helps them set up their accounts and connect to their data. We've also added a new product walkthrough to help users understand the core functionality within Budibase.

![Onboarding](https://user-images.githubusercontent.com/49767913/217481040-050592c8-8c34-4d44-8d26-95f446cb96ca.png)

---

### 🎨 New portal design
The new portal design makes it easier to navigate apps, users,  settings, and more.

![New portal](https://user-images.githubusercontent.com/49767913/217481377-472a5eba-5078-4647-b74e-48f9e4116a07.png)

---

### ❓New help button
We've redesigned the Budibase help button and made it easier to access the correct support channel when you need it.

![new help button](https://user-images.githubusercontent.com/49767913/217481787-2b2008ed-2fae-453f-8c64-75b61db3335a.png)

---

### Additional new improvements 
- Attachments are only accessible over pre-signed URLs. This improves security and ensures only users with the right access levels can access the attachment.

---

### Bugs

Handlebars incorrect syntax validation message #8329
Org logo is broken in new installs / tenants #9444 
Filter or selection is not applied when exporting data #8766 
Create column internal DB issues #9011
Alphabetical option not working on options picker #8992
Incorrect looping array binding will cause test to crash for external SQL database query rows #8989
Data provider with a form relationship as a source does not work #8910
In-app user sync improvements #4900
Option picker does not follow ordering from a custom query #8961
When using a formula column as the display column, the table will throw an error and not load data until you set a Sort By column #9377
MySQL Date/Time field search field in Table block throws error #9380
Unable to deactivate Google/OIDC auth when the other fields are empty #9351
Changing the table in a form block can cause incorrect field type placeholder warnings #9217
Ejecting a block removes filter  #9385
Cannot immediately navigate to Data section after adding component in Design #9358
Table clipping options picker #9080
Dragging a button out of a table can cause the page to error and require a page refresh #8767
Queries: results and transformer aren't reset while switching between queries #8852
Deleting User from Main Users Table Does Not Delete App User Row #7220
Incrementing hour with time only Date Picker #9138
MySQL BIT datatype checkbox in form cannot be saved #9215
Deleting automation and publishing app did not remove automation from test production environment #7865
Table Views: Unable to create a View #9141
Table Filtering Issues #8947
Components: Cannot Edit Component Name #8946
Very long screen names cause side panel to be unaccessible #8950

---

Thanks for reading. Please let us know what you think in the comments below. 

**Budibase team**

**Comment by NicolasFabre81** [comment]
Thanks for this ! What about the https://github.com/Budibase/budibase/discussions/9041 , is it corrected ?
